site/content/en/docs/faq/_index.md

```diff
@@ -82,3 +83,12 @@ Simply run the following command to be enrolled into beta notifications.
 ```
 minikube config set WantBetaUpdateNotification true
 ```
+
+## Can I remove/disable the emojis in minikube ?
+
+Yes ! if you reallly dislike the emoji :( you could set MINIKUBE_IN_STYLE envioronment variable to disable the emojis
```

Comment:
Yes! If you prefer not having emoji in your minikube output, just set the `MINIKUBE_IN_STYLE` environment variable to `0` or `false`:

Reply:
done thanks